src/modules/tracker.js

```diff
@@ -454,7 +454,7 @@ class Tracker {
         bodyParams.num_results_viewed = num_results_viewed;
       }
 
-      this.requests.queue(url, 'POST', applyParams(bodyParams, this.options));
+      this.requests.queue(`${url}${applyParamsAsString({}, this.options)}`, 'POST', bodyParams);
```

Comment:
Doesn't seem right. Inherited properties from constructor aren't being applied to bodyParams since we took applyParams out (`ui`, `key`, `us`, etc.).

Running the tests should reveal more detail.

Reply:
We're also missing a `?` here. This will generate something like: `https://ac.cnstrc.com/v2/behavioral_action/browse_result_clickc=cio-mocha&i=420334ed-a726-4f3b-ac8f-4d3b49c643c8&s=1&key=ZqXaOfXuBWD4s3XzCI1q&_dt=1574280607096`

Reply:
YAAAAAAAS!

Reply:
Test don't work, the PR is still a work in progress